graphql/src/main/java/com/scalar/db/graphql/GraphQlFactory.java

```diff
@@ -119,6 +118,7 @@ private GraphQLCodeRegistry createGraphQLCodeRegistry(
             new QueryScanDataFetcher(storage, helper));
       }
     }
+    builder.dataFetcher(coordinates(mutationObjectType, "abort"), new MutationAbortDataFetcher());
```

Comment:
Don't we need to do this:
```suggestion
    if (transactionManager != null) {
      builder.dataFetcher(coordinates(mutationObjectType, "abort"), new MutationAbortDataFetcher());
    }
```

Reply:
Good point! I missed that. Applied the change and arranged tests for this point.